### ISSUE TITLE
Don't load plugins we're not using

### ISSUE THREAD

**Issue by brendanlong** [jangernert/FeedReader#574]
It looks like one of the recent changes has caused us to load all plugins, and some of the do things on init:

```
$ feedreader 
** INFO: FeedReader 2.2-dev
(feedreader:28459): Gtk-DEBUG: Connecting to session manager
** (feedreader:28459): DEBUG: MainWindow: setupCSS
** (feedreader:28459): DEBUG: backend: constructor
** (feedreader:28459): DEBUG: backend: new FeedServer and login
** (feedreader:28459): DEBUG: feedserver: engine load Local RSS Backend
** (feedreader:28459): DEBUG: feedserver: plugin loaded Local RSS Backend
** (feedreader:28459): DEBUG: feedserver: engine load Tiny Tiny RSS
** (feedreader:28459): DEBUG: feedserver: plugin loaded Tiny Tiny RSS
** (feedreader:28459): DEBUG: feedserver: engine load OwnCloud News
** (feedreader:28459): DEBUG: feedserver: plugin loaded OwnCloud News
** (feedreader:28459): DEBUG: feedserver: engine load feedly
** (feedreader:28459): DEBUG: feedserver: plugin loaded feedly
** (feedreader:28459): DEBUG: feedserver: engine load Feedbin
** (feedreader:28459): DEBUG: feedserver: plugin loaded Feedbin
** (feedreader:28459): DEBUG: feedserver: engine load TheOldReader
** (feedreader:28459): DEBUG: feedserver: plugin loaded TheOldReader

** (feedreader:28459): WARNING **: Utils.gsettingReadString: failed to read org.gnome.feedreader.oldreader username

** (feedreader:28459): WARNING **: Utils.gsettingReadString: failed to read org.gnome.feedreader.oldreader access-token

** (feedreader:28459): WARNING **: invalid type for attribute type

** (feedreader:28459): WARNING **: Utils.gsettingReadString: failed to read org.gnome.feedreader.oldreader username

** (feedreader:28459): WARNING **: oldReaderUtils.getPassword: could not load password
** (feedreader:28459): DEBUG: feedserver: engine load freshRSS
** (feedreader:28459): DEBUG: feedserver: plugin loaded freshRSS
** (feedreader:28459): DEBUG: feedserver: engine load InoReader
** (feedreader:28459): DEBUG: feedserver: plugin loaded InoReader

** (feedreader:28459): WARNING **: Utils.gsettingReadString: failed to read org.gnome.feedreader.inoreader username

** (feedreader:28459): WARNING **: Utils.gsettingReadString: failed to read org.gnome.feedreader.inoreader access-token
** INFO: Plugin Name: Feedbin
** INFO: Plugin Version: 0.1
** INFO: Plugin Website: http://jangernert.github.io/FeedReader/
** INFO: Plugin Dir: /usr/local/lib64/FeedReader/plugins/
** (feedreader:28459): DEBUG: backend: login status = FEED_READER_LOGIN_RESPONSE_SUCCESS
```

**Comment by jangernert** [comment]
Hm, I see. If you are facing the login page we now have to load every plugin, to put them in the list of available options. But once you are using FeedReader with only that one plugin that is not neccesary any more. However we need to reload them when the user logs out and wants to log back in again.

**Comment by bil-elmoussaoui** [comment]
@jangernert I have an idea how to deal with this, i will give it a try right now 👍

**Comment by jangernert** [comment]
fixed by #581
